libgphoto2_port/gphoto2/gphoto2-port-log.h

```diff
@@ -218,6 +218,19 @@ __attribute__((__format__(printf,4,5)))
   char*
   gpi_vsnprintf (const char* format, va_list args);
 
+static inline char*
+aprintf(const char *fmt, ...)
```

Comment:
That would have been nice to add the attributes to mark this as a "format" https://gcc.gnu.org/onlinedocs/gcc-13.3.0/gcc/Common-Function-Attributes.html - then the compiler would be able to give warnings. Without this it introduce more possibilities of error.

Reply:
will add that. thanks for the hint.